app/src/cpp/cfc-cpp/CMakeLists.txt

```diff
@@ -23,3 +23,5 @@ target_link_libraries(cfc-cpp
         ${OPENCV_LIBS}
         ${log-lib}
 )
+
+target_compile_options(cfc-cpp  PUBLIC "-DZSTD_STATIC_LINKING_ONLY")
```

Comment:
The start of the new changes. This is the flag we need for the zstd skippable frame apis, which we use for passing (in this case decoding) the suggested filename.